keyvi/src/cpp/dictionary/fsa/automata.h

```diff
@@ -40,6 +40,7 @@
 #include "dictionary/util/endian.h"
 #include "dictionary/fsa/internal/intrinsics.h"
 #include "dictionary/dictionary_merger_fwd.h"
+#include "dictionary/util/keyvi_file.h"
```

Comment:
Hmmm, isn't automata just one part of `keyvi` ? 

Meaning `keyvi_file = meanifest + automata + value_store`